packages/kmod-5.10-nvidia/kmod-5.10-nvidia.spec

```diff
@@ -0,0 +1,214 @@
+%global nvidia_tesla_470_version 470.82.01
+%global spdx_id %(bottlerocket-license-tool -l $PWD/rpmbuild/BUILD/Licenses.toml spdx-id nvidia)
+%global license_file %(bottlerocket-license-tool -l $PWD/rpmbuild/BUILD/Licenses.toml path nvidia -p ./licenses)
+
+Name: %{_cross_os}kmod-5.10-nvidia
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: NVIDIA drivers for the 5.10 kernel
+# We use these licences because we only ship our own software in the main package,
+# each subpackage includes the LICENSE file provided by the run scripts
+License: Apache-2.0 OR MIT
+URL: http://www.nvidia.com/
+
+# NVIDIA .run scripts from 0 to 199
+Source0: https://us.download.nvidia.com/tesla/%{nvidia_tesla_470_version}/NVIDIA-Linux-%{_cross_arch}-%{nvidia_tesla_470_version}.run
```

Comment:
nit: it is considered bad form to conditionally include files like this in the spec file since then a source RPM won't be complete - whether it includes the x86 or arm64 run file will depend on what it was built for rather than what the end user of the srpm wants to build for.

In the context of our distro, it's fine since we don't build source RPMs, but I'd still prefer to include both `.run` files and conditionally invoke the right one, just as a stylistic twitch.